ui/ui-frontend/projects/archive-search/src/app/archive/archive-preview/archive-unit-objects-details-tab/archive-unit-objects-details-tab.component.ts

```diff
@@ -110,26 +123,28 @@ export class ArchiveUnitObjectsDetailsTabComponent implements OnChanges, OnInit
     const headers = new HttpHeaders().append('Content-Type', 'application/json').append('X-Access-Contract-Id', this.accessContractId);
     this.archiveService.getObjectById(archiveUnit['#id'], headers).subscribe({
       next: (unitObject) => {
-        this.unitObject = unitObject;
-        this.versionsWithQualifiersOrdered = qualifiersToVersionsWithQualifier(this.unitObject['#qualifiers']);
+        this.archiveUnit.objectGroup = unitObject;
+        this.archiveUnit.objectGroup.versionsWithQualifiers = qualifiersToVersionsWithQualifier(unitObject['#qualifiers']);
+
+        this.unitObject = this.archiveUnit.objectGroup;
+        this.versionsWithQualifiersOrdered = this.archiveUnit.objectGroup.versionsWithQualifiers;
```

Comment:
Pourquoi dupliquer l'information dans objectGroup et dans this.unitObject et this.versionsWithQualifiersOrdered ?

Reply:
c'etait pour ne pas apporter trop de changements       
je vais retirer unitObject et versionsWithQualifiersOrdered

Reply:
Ah. OK. Y'a du backport à faire ?

Reply:
non c'est v8 ce bug